server/utilities/winston_logger.js

```diff
@@ -0,0 +1,32 @@
+require('dotenv').config()
+const { format, createLogger, transports } = require('winston')
+const { timestamp, combine, errors, json, colorize, printf } = format
+
+const logger = createLogger({
+  format: combine(timestamp(), errors({ stack: true }), json()),
+  transports: [new transports.File({ filename: 'error.log' })]
+})
+
+// create log format for when we're in development
+const logFormat = printf(({ level, message, timestamp, stack }) => {
+  return `${timestamp} ${level}: ${stack || message}`
+})
+
+// if we're not in production, then log to the console
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Nice work on this setup! In the future, maybe we should consider setting a separate environment variable for logging output to give us some more flexibility.